### PR TITLE
setting the overall score in validators

### PR DESCRIPTION
**Pull request by muku314115**
## 1. Summary
Fixes # (issue)
built on https://github.com/ingenuity-build/quicksilver/pull/457 
Test cases ensure types.Validator.score == icstypes.Validator.score

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

File: x/participationrewards/keeper/rewards_lockup.go [hunk 1]
```diff
@@ -2,12 +2,17 @@ package keeper
...
+	"github.com/coinbase/rosetta-sdk-go/storage/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/ingenuity-build/quicksilver/x/participationrewards/types"
 )
 
 func (k Keeper) AllocateLockupRewards(ctx sdk.Context, allocation math.Int) error {
+	if allocation.IsNegative() {
+		k.Logger(ctx).Error("invalid allocation requested", "allocation", allocation)
+		return errors.ErrInvalidValue
```
Comment (aljo242) [inline x/participationrewards/keeper/rewards_lockup.go:14]: Why use this error type from the `rosetta-sdk` pkg?

**Comment by aljo242** [comment]
just gotta run `make format`

**Review by joe-bowman — APPROVED** [review]
lgtm

**Review by ajansari95 — COMMENTED** [review]
conflicts

**Review by joe-bowman — APPROVED** [review]
Lgtm

**Review by ajansari95 — APPROVED** [review]
lgtm

**Pull request merged**